bot_interface.py

```diff
@@ -140,6 +146,7 @@ def clear_previous_messages(user_id):
 @bot.message_handler(commands=['start'])
 def start(message):
     user_id = message.chat.id
+    logger.info(f"Бот запущен пользователем {user_id}")
```

Comment:
there will probably be too many similar messages and should be done logger.debug instead .info (logger.debug will include all info logs if there is an env variable DEBUG=1 - present)